server/controllers/users/adminSignup.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-trailing-spaces */
+import express from 'express';
+import bcrypt from 'bcrypt';
+import Joi from '@hapi/joi';
+import client from '../../config/config';
+
+
+const router = express.Router();
+
+router.post('/adminSignup', (req, res) => {
```

Comment:
/adminSignup is a very predictable name and since your repo is public it is visible to everybody, change it to something secret, use a name from your environment. Eg. `/${process.env.SECRET_ROUTE}`. The value of `SECRET_ROUTE` can be any unpredictable jargon you like.